health/health.d/net.conf

```
@@ -101,7 +105,9 @@ component: Network
     every: 1m
      warn: $this >= 2
     delay: up 1m down 1h multiplier 1.5 max 2h
-     info: ratio of inbound dropped packets for the network interface $family over the last 10 minutes
+     info: Ratio of inbound dropped packets for the network interface $family over the last 10 minutes. \
+           Network interface $family dropped many packets. \
+           It means packets received but not processed, e.g. due to lack of resources or unsupported protocol.
```

```suggestion
           It means packets were received but not processed, e.g. due to lack of resources or unsupported protocol.
```

Without the use of ‘were’, it’s ambiguous whether ‘packets’ is the subject or object of ‘received’, which sounds very unnatural to native speakers.